xarray/core/computation.py

```diff
@@ -893,7 +893,7 @@ def apply_ufunc(
     dataset_fill_value: object = _NO_FILL_VALUE,
     keep_attrs: bool | str | None = None,
     kwargs: Mapping | None = None,
-    dask: str = "forbidden",
```

Comment:
I see `keep_attrs` and `dataset_join` doesn't have correct either if you want to fix those too.

Reply:
Yeah, I do find that merging the small PR makes it more likely I'll make a PR when I spot something. Hope that's reasonable to folks. (and totally reasonable to make the suggestion)